server-config.json

```diff
@@ -8,6 +8,7 @@
     "rskNodeUrl": "http://rskj:4444",
     "devMode": true,
     "customReplenish": false,
+    "allowForSponsoredTx": false,
```

Comment:
I think that you were right when you mentioned that we should allow for sponsored transactions by default
```suggestion
    "allowForSponsoredTx": true,
```

Reply:
Or maybe, since it's default value is true, we could just remove this parameter.

Reply:
you're right, thanks. i've changed it to true but still left it there; i prefer having this field explicit in the config even if the end result is the same